### PR TITLE
Makes offline test use evaluateAsync

### LINKED ISSUE #266: Update offline gatherer to use `evaluateAsync`
Sam, you okay to take this?

### PR DESCRIPTION
**Pull request by paullewis**
Fixes #266 - @samccone figured you were super busy with stuff, and I was getting wacky results from the existing test so I thought I'd just get it sorted. Hope that's okay.

### CODE REVIEW

File: src/gatherers/offline.js [hunk 1]
```diff
@@ -55,69 +55,25 @@ class Offline extends Gather {
...
-      }).then(_ => {
-        this.artifact = {responseCode};
-      });
-    });
+    return Offline
+        .goOffline(driver)
+        .then(_ => {
+          return driver
+              .evaluateAsync(`(${requestPage.toString()}())`)
+              .then(responseCode => {
```
Comment (samccone) [inline src/gatherers/offline.js]: change param to offlineResponseCode

File: src/gatherers/offline.js [hunk 2]
```diff
@@ -55,69 +55,25 @@ class Offline extends Gather {
...
+                this.artifact = {
```
Comment (samccone) [inline src/gatherers/offline.js]: `this.artifact = {offlineResponseCode}`

**Comment by samccone** [comment]
1 nit and then +1

**Comment by samccone** [comment]
Thanks @paullewis 💃

File: src/gatherers/offline.js (hunk 2 above)
Comment (paullewis) [inline src/gatherers/offline.js]: Done.

File: src/gatherers/offline.js (hunk 1 above)
Comment (paullewis) [inline src/gatherers/offline.js]: Done.

File: src/gatherers/offline.js [hunk 3]
```diff
@@ -55,69 +55,23 @@ class Offline extends Gather {
...
+        .then(_ => {
+          return driver
+              .evaluateAsync(`(${requestPage.toString()}())`)
+              .then(offlineResponseCode => {
+                this.artifact = {offlineResponseCode};
+              });
+        })
+        .then(_ => {
+          return Offline.goOnline(driver);
+        });
```
Comment (brendankenny) [inline src/gatherers/offline.js]: can flatten this promise chain a bit, like
```
return Offline.goOffline(driver)
  .then(_ => {
    return driver.evaluateAsync(`(${requestPage.toString()}())`)
  })
  .then(offlineResponseCode => {
    this.artifact = {offlineResponseCode};
  })
  .then(_ => {
    return Offline.goOnline(driver);
  });```

Comment (paullewis) [inline src/gatherers/offline.js]: Done.

**Comment by brendankenny** [comment]
LGTM!

**Pull request merged**